dist/angular-ui-tree.js

```diff
@@ -628,11 +628,6 @@
               firstMoving = true;
               dragInfo = UiTreeHelper.dragInfo(scope);
 
-              // Fire dragStart callback
-              scope.$apply(function () {
-                scope.$treeScope.$callbacks.dragStart(dragInfo.eventArgs(elements, pos));
```

Comment:
because in this line I am getting `elements=undefined` and `pos=undefined` as they are getting set in the below lines. So this callback must be called after setting up the variables.